tests/acceptance/features/apiSpaces/spaceDownload.feature

```diff
@@ -0,0 +1,22 @@
+@api
+Feature: Download space
+  As a user
+  I want to be able to download space
+  So that I can store it in my local storage
+
+
+  Scenario: user downloads a space
+    Given user "Alice" has been created with default attributes and without skeleton files
+    And using spaces DAV path
+    And the administrator has assigned the role "Space Admin" to user "Alice" using the Graph API
```

Comment:
can space admin only download space? if not maybe we can add other scenarios too in the next PR